src/views/TechnicalAssistance/Homepage.tsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * @file "Technical Assistance" (TRB) homepage.
+ * Created with reference to `src/views/MakingARequest`, `src/views/MyRequests/Table`.
+ */
+
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useRouteMatch } from 'react-router-dom';
+import {
+  Column,
+  useFilters,
+  useGlobalFilter,
+  usePagination,
+  useSortBy,
+  useTable
+} from 'react-table';
+import { useQuery } from '@apollo/client';
+import { Button, Link, Table } from '@trussworks/react-uswds';
+
+import MainContent from 'components/MainContent';
+import PageHeading from 'components/PageHeading';
+import PageLoading from 'components/PageLoading';
+import GlobalClientFilter from 'components/TableFilter';
+import TablePageSize from 'components/TablePageSize';
+import TablePagination from 'components/TablePagination';
+import TableResults from 'components/TableResults';
+import GetTrbRequestsQuery from 'queries/GetTrbRequestsQuery';
+import {
+  GetTrbRequests,
+  // eslint-disable-next-line camelcase
+  GetTrbRequests_trbRequestCollection
+} from 'queries/types/GetTrbRequests';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon
+} from 'utils/tableSort';
+import NotFound from 'views/NotFound';
+import { formatDate } from 'views/SystemProfile';
+
+function Homepage() {
+  const { t } = useTranslation('technicalAssistance');
+  const { url } = useRouteMatch();
+
+  const { loading, error, data } = useQuery<GetTrbRequests>(
+    GetTrbRequestsQuery
+  );
+
+  const trbRequests = data?.trbRequestCollection || [];
+
+  // eslint-disable-next-line camelcase
+  const columns = useMemo<Column<GetTrbRequests_trbRequestCollection>[]>(() => {
+    return [
+      {
+        Header: t<string>('table.header.requestName'),
+        accessor: 'name'
+      },
+      {
+        Header: t<string>('table.header.submissionDate'),
+        accessor: 'createdAt',
+        Cell: ({ value }) => formatDate(value)
+      },
+      {
+        Header: t<string>('table.header.status'),
+        accessor: 'status'
+      }
+    ];
+  }, [t]);
+
+  const {
+    canNextPage,
+    canPreviousPage,
+    getTableBodyProps,
+    getTableProps,
+    gotoPage,
+    headerGroups,
+    nextPage,
+    pageCount,
+    pageOptions,
+    prepareRow,
+    previousPage,
+    rows,
+    setGlobalFilter,
+    setPageSize,
+    state
+  } = useTable(
+    {
+      columns,
+      data: trbRequests,
+      autoResetSortBy: false,
+      autoResetPage: false,
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'name', desc: false }], []),
+        pageIndex: 0,
+        pageSize: 10
+      }
+    },
+    useFilters,
+    useGlobalFilter,
+    useSortBy,
+    usePagination
+  );
+
+  if (loading) {
+    return <PageLoading />;
+  }
+
+  if (error) {
+    return <NotFound />;
+  }
+
+  return (
+    <MainContent className="technical-assistance grid-container">
+      <PageHeading>{t('heading')}</PageHeading>
+
+      <Link
```

Comment:
This should be `USWDSReactLink` instead of Truss' `Link`.  Truss doesn't utilize react-router, instead uses an `<a />` tag which cause a browser refresh